### PR TITLE
Hard-coded the %appdata% paths

### PR DESCRIPTION
**Pull request by n8ventures**
Hard-coded the data directory (`%appdata%` for Windows, `$HOME/Library/Application Support` for MacOS) to the program so you only need to put one argument which is specifically the save location of the fonts.

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -1,9 +1,10 @@
 [package]
-name = "adobe-font-grabber"
-version = "0.1.1"
+name = "adode-font-grabber"
```
Comment (bennyboer) [inline Cargo.toml]: I think you have a typo here 😄

**Review by bennyboer — CHANGES_REQUESTED** [review]
Only found the one typo in the crate name, otherwise seems like a useful addition! Thanks for contributing!

File: Cargo.toml (hunk 1 above)
Comment (n8ventures) [reply Cargo.toml]: My bad! But, thank you!

Comment (n8ventures) [reply Cargo.toml]: Fixed the typo!

**Comment by bennyboer** [comment]
Thanks again! I'll provide a new release build for Windows the next few days 👍

**Pull request merged**